meson.build

```diff
@@ -93,10 +93,11 @@ conf_data.set('XNVME_BE_MACOS_ENABLED', is_darwin)
 conf_data.set('XNVME_BE_FBSD_ENABLED', is_freebsd)
 
 conf_data.set('XNVME_BE_SPDK_ENABLED', (is_linux or is_freebsd) and get_option('with-spdk'))
-conf_data.set('XNVME_BE_SPDK_TRANSPORT_PCIE_ENABLED', (is_linux or is_freebsd) and get_option('with-spdk'))
+conf_data.set('XNVME_BE_SPDK_TRANSPORT_PCIE_ENABLED', (is_linux or is_freebsd or is_windows) and (get_option('with-spdk') or get_option('with-spdk-win')))
 conf_data.set('XNVME_BE_SPDK_TRANSPORT_TCP_ENABLED', (is_linux or is_freebsd) and get_option('with-spdk'))
 conf_data.set('XNVME_BE_SPDK_TRANSPORT_RDMA_ENABLED', (is_linux or is_freebsd) and get_option('with-spdk'))
 conf_data.set('XNVME_BE_SPDK_TRANSPORT_FC_ENABLED', (is_linux or is_freebsd) and get_option('with-spdk'))
+conf_data.set('XNVME_BE_SPDK_WIN_ENABLED', (is_windows) and get_option('with-spdk-win'))
```

Comment:
I think you do not need an additional option? How about just re-using ``with-spdk``?

Reply:
Would this:
``
XNVME_BE_SPDK_WPDK_ENABLED
``
be preferable to:
``
XNVME_BE_SPDK_WIN_ENABLED
``
?